geomesa-tools/conf/geomesa-env.sh

```diff
@@ -28,6 +28,7 @@ function setvar() {
     export $1=$2
   fi
 }
+GEOMESA_TOOLS_VERSION=%%gmtools.version%%
```

Comment:
I'm unsure if this is the best place for the version control. Let me know if you have a better idea.